src/ripple/rpc/handlers/LedgerRequest.cpp

```diff
@@ -120,10 +120,12 @@ Json::Value doLedgerRequest (RPC::Context& context)
     }
 
     // Try to get the desired ledger
+    // Verify all nodes even if we think we have it
     auto ledger = context.app.getInboundLedgers().acquire (
         ledgerHash, ledgerIndex, InboundLedger::fcGENERIC);
 
-    if (context.app.config().standalone())
+    // In standalone mode, accept the ledger from the ledger cache
```

Comment:
Alternate suggestion:
// If not in the node store and in standalone mode
// accept the ledger from the ledger cache